test/TestExpectations.json

```diff
@@ -1691,6 +1709,12 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[drag-and-drop.spec] Drag n' Drop should drag and drop",
```

Comment:
Why is this failing?

Reply:
Because an extra `dragover` occurs with CDP. This distinction is important; it implies CDP will trigger an event twice in the same coordinate.